internal/provider/resource_gitlab_project_share_group_test.go

```diff
@@ -109,6 +109,34 @@ func testAccCheckGitlabProjectIsNotShared(projectName string) resource.TestCheck
 	}
 }
 
+func testAccCheckGitlabProjectShareGroupDestroy(s *terraform.State) error {
+	var projectId string
+	var groupId int
+	var err error
+
+	for _, rs := range s.RootModule().Resources {
```

Comment:
In case there is no resource, the subsequent `GetProject` will fail ...

Reply:
Fixed